.gitignore

```diff
@@ -2,4 +2,7 @@
 venv/
 
 # Python runtime files
-**/__pycache__/
\ No newline at end of file
+**/__pycache__/
+
+# Config file
+docker-compose.yaml
```

Comment:
Généralement le docker compose  c’est Le fichier qu’on veut partager dans toute l’équipe, faut plutôt le commit :)

Reply:
C'est ce que j'avais fait au début ! 
Mais comme je voulais simplifier le changement de port du back, et que le port doit être noté dans le docker-compose, j'ai pas envie que Git détecte un changement dans le fichier dès que quelqu'un change le port...
Tu en penses quoi ?